### PR TITLE
updated ambari admin password as part of ambari server setup

### PR DESCRIPTION
**Pull request by rashidaligee**
added functionality to update ambari admin password. refactored code to remove rubocop's warnings.

### CODE REVIEW

File: cookbooks/ambari/attributes/default.rb [hunk 1]
```diff
@@ -40,7 +40,8 @@
 
 
 node.default['ambari']['admin']['user'] = 'admin'
-node.default['ambari']['admin']['password'] = 'admin'
+node.default['ambari']['admin']['password'] = 'password'
+default['ambari']['admin']['default_password'] = 'admin'
```
Comment (http-418) [inline cookbooks/ambari/attributes/default.rb]: Is there a reason some of these are `node.default` and others are just `default` ?

Comment (rashidaligee) [reply cookbooks/ambari/attributes/default.rb]: I have used `node.default` in ambari recipe but it is suggested that `default` is the style, used in chef-bach code. I have started using `default` and @amithkanand have replaced `node.default` with `default` as part of pull request #1182.

**Comment by RiteshVajariya** [comment]
update `.to_s` wherever appropriate.

File: cookbooks/ambari/libraries/utils.rb [hunk 2]
```diff
@@ -16,12 +16,57 @@
...
-    return false
+  Chef::Log.info('View seems not created')
+  return false
+end
+
+def update_default_ambari_admin_password(number_retry: 0, max_retries: 24, delay: 5)
+  number_retry += 1
+  ambari_base_url = node['ambari']['ambari_server_base_url']
+  view_list_url = 'api/v1/views'
+  psswd_update_url = 'api/v1/users/username'
```
Comment (RiteshVajariya) [inline cookbooks/ambari/libraries/utils.rb]: Should `psswd_update_url` be as `passwd_update_url`?

File: cookbooks/ambari/libraries/utils.rb [hunk 3]
```diff
@@ -16,12 +16,57 @@
...
+  default_password = node['ambari']['admin']['default_password']
+  admin_password = node['ambari']['admin']['password']
+  admin_user = node['ambari']['admin']['user']
+
+  passwd_update_json = { 'Users' =>
```
Comment (RiteshVajariya) [inline cookbooks/ambari/libraries/utils.rb]: Should `passwd_update_json` be as `password_update_json`?

**Pull request merged**